src/client.rs

```diff
@@ -800,9 +799,10 @@ where
                     Err(_) => {
                         warn!("poll: failed to memoize features");
                     }
+                },
+                Err(err) => {
+                    warn!("poll: failed to retrieve features {}", err);
```

Comment:
This log was super useless, as it gave 0 information on why this failed. Related issue: https://github.com/Unleash/unleash-client-rust/issues/35